### ISSUE TITLE
Ideas for the distant future

### ISSUE THREAD

**Issue by proffapt** [metakgp/gyfe#25]
Following are the ideas for the distant future, come to them after the extraction / electives fetching logic is solid.
 
- [ ] Show tentative timetable - how will it look if the subject is selected
      Display users' current time table in the UI and below it the list of available electives.
      As the user selects the time table gets updated. It can serve as a demo for how their schedule 
      will look like
- [ ] Apply directly from the interface

Basically, let the user forget that laggy ERP interface and enjoy our beautiful wrapper for it :)

**Comment by suryaprakash0010** [comment]
please assign this issue to me .
